docs/source/quick_start.rst

```diff
@@ -0,0 +1,160 @@
+Quick Start
+============
+
+This guide is intended to jump start new Kokkos users (and beginners, in particular).  Kokkos Core is the foundation for all other Kokkos Ecosystem projects, and is readily usable in your own for performance portability.
+
+
+Download Latest and Build 
+-----------------------------
+
+.. note::
+
+  Please become familiar with `Kokkos Requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_, and verify that your machine has all necessary compilers, backend SDK and build system components.
```

Comment:
What about 

Please ensure that your system meets the `minimum requirements <https://kokkos.org/kokkos-core-wiki/requirements.html>`_ (cmake and compiler toolchain)

Reply:
Ping.  "[V]erify that your machine has all necessary compilers, backend SDK and build system components" is somewhat cryptic.